### DISCUSSION TITLE
DeSo.js and Code Examples

### DISCUSSION THREAD

**Discussion by 1dolinski** [deso-protocol/dips#217] [Medium CIPs]
Couldn't find a DIP mentioning DeSo.JS so here is one! The DIP is a library to make implementing deso auth easier and code examples for common frontend libraries (React/Angular/Vue)

**Core Team is prioritizing it**
https://bitclout.com/posts/18c8e56929fa242fee837f6496010a82b6a410c9e92d047b595df723cde5d770

<img width="528" alt="image" src="https://user-images.githubusercontent.com/1243216/153645834-68e5bf7f-e91b-4fc5-b453-3b658d8f7bec.png">

**Existing Community Implementations**
https://bitclout.com/posts/f1a19a3341d943b86395bcdd046f34222f4eb173b23065afa334c67515f16e82
https://bitclout.com/posts/270f90b83c14f62d8cb01e164b2656952cb158995a1cf771dc0861b1beb87528

**Community interest**
https://bitclout.com/posts/6391c14d4f05748c0658247face18ee1249a098f8e0b6d9fbb2ed94bf8054a77
https://bitclout.com/posts/46a801275644f6153dc13e345c119667ee38214eb65b07250c26aaa93cb804ec
https://bitclout.com/posts/43a8c3a614e9830e47d955dab8b030987f08761b2491151390240a54feb8cf05
https://bitclout.com/posts/774f830d16ba88a71b7e4bfe9c7eac0db1ba51cc962b4aabb4a3894289e14d64

**Comment by brootle** [comment]
@diamondhands0 what's the progress on Deso.js? I really need it. I am sure the number of apps that run on deso protocol will explode as soon as there is easy to use library.

**Reply by StarGeezerPhil** [reply]
Echo what's been said above. 

You have (hundreds?) of devs reproducing the exact same wheel to implement identity - all wasted time that could be on productive/value-add development and functionality for the platform.

This would be a huge catalyst for expediating development on deso.

**Comment by brootle** [comment]
@dharmeshs you want to join the discussion?
